.github/workflows/dev-docs-publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Dev docs publisher
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "*"
+    branches:
+      - main
+
+env:
+  DOCUMENTATION_CNAME: 'mapdl.docs.pyansys.com'
+
+jobs:
+  upload_dev_docs:
+    name: "Upload dev documentation"
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy the latest documentation
+        uses: pyansys/actions/doc-deploy-dev@v2
+        with:
+          cname: ${{ env.DOCUMENTATION_CNAME }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This action does not work as you expect... you need to have a "documentation-html" artifact for this to work. This means that you need to build the docs in the same workflow.

If you have doubts let me know.

As it is, this workflow will try to upload a non-existing artifact =)

Reply:
that is why I did later https://github.com/pyansys/pymapdl/pull/1701

now I wonder what is better, if rebuild the docs or just reuse the artifacts from the other workflow.

Reply:
Ha yeah... Didn't see that.

Your choice I guess. Reusing artifacts from other workflows is "weird" for me - but acceptable if that's what you want 😄

Reply:
I'm see if I can... and then decide xD